### PR TITLE
feat(bsky): update selectors for current version

### PR DESCRIPTION
**Pull request by RoootTheFox**
light mode is mainly untested/untouched, not fully finished yet
may have some issues because i changed a couple things quite a bit

i also want to restructure some parts of the style, but i might do that in another PR

## 🔧 What does this fix? 🔧

fixes some selectors for the current bluesky web app as i haven't touched this style in like, 6 months oops lol
(mostly buttons, backgrounds, etc being broken, and a lot of selectors no longer matching anything at all)
also themes direct messages now as those didn't exist when i last touched this style

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/bsky/catppuccin.user.css [hunk 1]
```diff
@@ -82,23 +85,66 @@
...
       }
     }
 
+    [style*="color: rgb(11, 15, 20)"] {
+      color: @text !important;
+
+      /* fix contrast issues */
+      & when not (@lookup = latte) {
+        color: @crust !important;
+      }
```
Comment (uncenter) [inline styles/bsky/catppuccin.user.css:104]:
```suggestion
      color: if(@lookup = latte, @text, @crust) !important;
```

Comment (RoootTheFox) [reply styles/bsky/catppuccin.user.css]: done :3

File: styles/bsky/catppuccin.user.css [hunk 2]
```diff
@@ -98,10 +98,8 @@
     [style*="color: rgb(11, 15, 20)"] {
       color: @text !important;
 
-      /* fix contrast issues */
-      & when not (@lookup = latte) {
-        color: @crust !important;
-      }
+      // fix contrast issues on latte
+      color: if(@lookup = latte, @text, @crust) !important;
```
Comment (uncenter) [inline styles/bsky/catppuccin.user.css]: You only need the latter `color` line here, the first one is redundant.

Comment (uncenter) [reply styles/bsky/catppuccin.user.css]: Of course the diff doesn't match what I'm looking at -_- but I mean the line with `color: @text !important;`, right before this "fix contrast issues on latte" comment.

Comment (RoootTheFox) [reply styles/bsky/catppuccin.user.css]: oh, oops. i meant to remove that lmao

**Review by uncenter — DISMISSED** [review]

**Review by isabelroses — CHANGES_REQUESTED** [review]
Could we try theming icons, should be something lik e `path[fill="hsl(211, 20%, 95.3%)"]`.

**Comment by RoootTheFox** [comment]
that was actually already implemented, i just forgot to update the selector ^^

**Review by isabelroses — APPROVED** [review]

**Comment by RoootTheFox** [comment]
maybe bit off-topic, but can you squash merge from the command line so i can retain my commit signatures?

**Pull request merged**